packages/opencensus-propagation-b3/src/b3Format.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Should be `Google LLC.`.

Reply:
Whoops, I see this has already been addressed. I was looking at a subset of the commits.

Reply:
Done. It was not for this file. Thanks.